pyproject.toml

```diff
@@ -8,4 +8,4 @@ build-backend = "setuptools.build_meta"
 [tool.pyright]
 include = ["src"]
 reportImportCycles = false
-reportUnnecessaryIsInstance = false
```

Comment:
Could we remove those changes from this PR? This makes the blame quite unreadable.